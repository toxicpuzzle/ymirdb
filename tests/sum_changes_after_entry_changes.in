SET a 1 2 3
SUM a
SET b 20
PUSH a b
SUM a
DEL b
POP a
BACKWARD b
SUM a
bye