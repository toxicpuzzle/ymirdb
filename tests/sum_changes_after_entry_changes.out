> ok

> 6

> ok

> ok

> 26

> not permitted

> b

> nil

> 6

> bye
